client/app/pages/tournament-settings/tournament-settings.component.html

```diff
@@ -10,27 +10,48 @@ <h3>Settings</h3>
-                        <div class="input-group-button">
-                            <button class="button" (click)="save('tournamentStage')"
-                                    [disabled]="settings['tournamentStage'].options == undefined"><i
-                                    class="fa fa-save"></i>&nbsp; Save
+                    <div class="grid-x grid-margin-x">
+                        <div class="cell medium-6">
+                            <h5>{{settings['tournamentStage'].display}}</h5>
+                            <div class="input-group">
+                                <select class="input-group-field" [(ngModel)]="settings['tournamentStage'].value"
+                                        [disabled]="settings['tournamentStage'].options == undefined">
+                                    <option *ngFor="let option of settings['tournamentStage'].options"
+                                            value="{{option}}">
+                                        {{option}}
+                                    </option>
+                                </select>
+                                <div class="input-group-button">
+                                    <button class="button" (click)="save('tournamentStage')"
+                                            [disabled]="settings['tournamentStage'].options == undefined"><i
+                                            class="fa fa-save"></i>&nbsp; Save
+                                    </button>
+                                </div>
+                            </div>
+
+                            <h5>Delete Tournament Data</h5>
+                            <label>Press Delete to delete Teams and Matches. <br/> Note: You must first delete all
+                                scoring data.</label>
+                            <button class="button alert" data-open="model-delete"
+                                    (click)="setDeleteModel(tournamentDataService)" [disabled]="!hasDataInDb">Delete
                             </button>
                         </div>
-                    </div>
+                        <div class="cell medium-6">
+                            <h5>Number of rounds</h5>
+                            <label>
+                                Set the default number of rounds.
+                                <br>
+                                This setting is used when you import a partial schedule file or a team list.
+                            </label>
 
-                    <h5>Delete Tournament Data</h5>
-                    <label>Press Delete to delete Teams and Matches. <br/> Note: You must first delete all scoring data.</label>
-                    <button class="button alert" data-open="model-delete"
-                            (click)="setDeleteModel(tournamentDataService)" [disabled]="!hasDataInDb">Delete
-                    </button>
+                            {{settings['practiceAmountOfMatches'].display}}
```

Comment:
Please keep note of the words you use. These are the number of **rounds**, not **matches** in each **stage**. Please use correct uniform naming.